File: Hello-everyone-/Java_Projects/TwoD.java
public class TwoD
   {
      public static void main(String[] args)
      {  int [][] arr={{  0,  4, 12 },
                       {  4,  0,  2 },
                       { 12,  2,  0 }
                       };
   	 int i,j;
         for(i=0;i<3;i++){
            for(j=0;j<3;j++){
             if(arr[i][j]!=arr[j][i]){
              System.out.println("Not symmetrical");
              return;}
                   }
               }
              System.out.println("Matix is symmetric");
       }   
      }

File: Hello-everyone-/C++_Projects/TwoD.cpp
#include<iostream>
using namespace std;

template<typename T>
void twoD(T arr[3][3], int n){
   int i, j;
   for(i=0;i<n;i++)
     for(j=0;j<n;j++)
        if(arr[i][j]!=arr[j][i]){
          cout<<"Not symmetric"<<endl;
          return;}
   cout<<"Matrix is symmetric"<<endl;
  }

int main(){
int a[3][3]={{1,0,0},
             {0,1,0},
             {0,0,1}} ;
twoD(a,3);
return 0;
}
